test/Makefile

```diff
@@ -16,7 +16,7 @@ TEST_DIRECTORY    := $(abspath $(dir $(CURRENT_MAKEFILE)))
 CLAR_PATH         := $(dir $(TEST_DIRECTORY))
 CLAR_FIXTURE_PATH := $(TEST_DIRECTORY)/resources/
 
-CFLAGS=-g -I.. -I. -Wall -DCLAR_FIXTURE_PATH=\"$(CLAR_FIXTURE_PATH)\"
+CFLAGS?=-g -I.. -I. -Wall -DCLAR_FIXTURE_PATH=\"$(CLAR_FIXTURE_PATH)\"
```

Comment:
@rsbeckerca Changing CFLAGS like this is somewhat orthogonal to the enablement of NonStop architectures. I also think that we need to be a bit more thorough here and not only make the CFLAGS overridable, but also split it up into CFLAGS and CPPFLAGS. The latter would then hold the `-DCLAR_FIXTURE_PATH` define such that overriding CFLAGS won't cause us to drop that necessary define.

Overall, I think we should move this change into a separate PR such that the enablement of NonStop in `clar/sandbox.h` can be merged faster.